CMakeCommon.cmake

```diff
@@ -330,8 +341,8 @@ endfunction ()
 function (ReadConfigJson)
     file (READ "${CMAKE_SOURCE_DIR}/config.json" json)
```

Comment:
There are repositories where we compile multiple Add-Ons from one source code. In this case there are multiple config.json files like `addon1_config.json` and `addon2_config.json`. Because of this it would be good to make the config json path configurable.

Reply:
#39